x/emissions/keeper/msgserver/msg_server_registrations.go

```diff
@@ -74,6 +78,10 @@ func (ms msgServer) Register(ctx context.Context, msg *types.MsgRegister) (*type
 
 // Remove registration from a topic
 func (ms msgServer) RemoveRegistration(ctx context.Context, msg *types.MsgRemoveRegistration) (*types.MsgRemoveRegistrationResponse, error) {
+	if msg == nil {
+		return nil, types.ErrReceivedNilRequest
+	}
+
```

Comment:
prob not needed + other places where you added this

Reply:
removed